fedimint-testing/src/bin/fixtures.rs

```diff
@@ -151,11 +151,12 @@ async fn await_fedimint_block_sync() -> anyhow::Result<()> {
 }
 
 async fn run_bitcoind() -> anyhow::Result<()> {
-    let btc_dir = env::var("FM_BTC_DIR")?;
-
-    // spawn bitcoind
+    let current_dir = env::current_dir().unwrap();
+    let conf_path = current_dir.join("misc/test/bitcoin.conf");
+    let conf_path_string = conf_path.into_os_string().into_string().unwrap();
+    //Spawn bitcoind
     let mut bitcoind = Command::new("bitcoind")
-        .arg(format!("-datadir={btc_dir}"))
```

Comment:
Why are we removing the data dir?

Reply:
Datadir had location to the temp directory.

Reply:
I don't think we need it if we already have -conf location

Reply:
Are you sure? I thought the configuration directory and the data directory are different things. Data directory is where the blocks are written to vs the configration directory is where to read the conf file from. If we're just changing where the conf file is read from, why does the data directory need to change?

Reply:
According to my knowledge when bitcoind is started, it reads its configuration file to determine the location of its data directory, among other settings. I may be wrong. So should I add `-datadir={cwd}/misc/test`?

Reply:
:eyes: how are tests passing?

is bitcoind assuming datadir to `~/.bitcoind`?

Reply:
I think yes, as it defaults to `~/.bitcoind` (in case of linux) when no `-datadir` is mentioned.

Reply:
do not remove `data_dir`, using ~/.bitcoind is not ideal

Reply:
No, don't change it.

Reply:
cool